src/views/clusteroverview/utils/VirtctlPopup.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+
+import { ConsoleCLIDownloadModelGroupVersionKind } from '@kubevirt-ui/kubevirt-api/console';
+import ExternalLink from '@kubevirt-utils/components/ExternalLink/ExternalLink';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { K8sResourceCommon, useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import { Button, Popover, PopoverPosition } from '@patternfly/react-core';
+
+import { VIRTCTL_DOWNLOADS } from './constants';
+import { K8sResourceKind } from './types';
+
+import './VirtctlPopup.scss';
+
+const VirtctlPopup: React.FC = () => {
+  const { t } = useKubevirtTranslation();
+  const [cliTools] = useK8sWatchResource({
+    groupVersionKind: ConsoleCLIDownloadModelGroupVersionKind,
+    isList: true,
+    namespaced: false,
+  });
+
+  const tools = Array.isArray(cliTools) ? cliTools : [...[cliTools]];
+  // skipcq: JS-0349
```

Comment:
what does it mean?

Reply:
does this skip DeepSource checks? why?

Reply:
@metalice @glekner Exactly, it's the equivalent of `eslint-disable`. It was throwing an error because of the `K8sResourceKind` cast. If I don't cast it I get a type error because `spec` isn't on `K8sResourceCommon`. I'm open to a better way to handle that.

Reply:
useK8sWatchResource<K8sResourceKind> won't help in this case?